OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -47,12 +47,12 @@ class CellTrigger
 			public readonly CPos[] Footprint;
 			public bool Dirty;
 
-			readonly Action<Actor> onActorEntered;
-			readonly Action<Actor> onActorExited;
+			readonly Action<Tuple<Actor, Player>> onActorEntered;
+			readonly Action<Tuple<Actor, Player>> onActorExited;
```

Comment:
Making OnExited run first just makes sense.  I may have had a specific bug-related reason for having it backwards, but lets hope not.

I don't see a strong usecase at this point for having the things that bind to triggers handle actor changes.  Run OnExited on the exiting actors, and then on any owner-changed actors that haven't left the region.  Then repeat the process for OnEntered.